src/openvic-simulation/GameManager.cpp

```diff
@@ -124,16 +132,23 @@ bool GameManager::expand_building(Province::index_t province_index, const std::s
 	return province->expand_building(building_type_identifier);
 }
 
+static constexpr colour_t LOW_ALPHA_VALUE = float_to_alpha_value(0.4f);
+static constexpr colour_t HIGH_ALPHA_VALUE = float_to_alpha_value(0.7f);
+static constexpr colour_t LAND_COLOUR = 0x0D7017;
+static constexpr colour_t WATER_COLOUR = 0x4287F5;
```

Comment:
Can you comment why these values please? I did google and found nothing obvious.